packages/jaspr/lib/src/foundation/constants.dart

```diff
@@ -71,7 +71,7 @@ const double precisionErrorTolerance = 1e-10;
 /// backed by the same kind of object. Thus a double `0.0` is identical
 /// to an integer `0`. This is not true for Dart code running in AOT or on the
 /// VM.
-const bool kIsWeb = identical(0, 0.0);
+const bool kIsWeb = bool.fromEnvironment('dart.library.js_util');
```

Comment:
Legacy interop libraries are set to be removed, so I think it's better to use `dart.library.js_interop` for this to work in the future.

Reply:
Done